tests/test_yamlCamera.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of obs_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import pkg_resources
+import unittest
+
+from lsst.obs.base.yamlCamera import makeCamera
+
+
+class YamlCameraTestCase(unittest.TestCase):
```

Comment:
I assume the idea is to flesh this out later with more tests of yaml camera? Do we have any tickets for that as part of the overall yaml camera work?

Reply:
I have no real idea about YAML camera. I'm doing the minimum possible to get a test so that I could use it in the visit definition test. I imagine that @czwa might have a plan since he's moving HSC to YAML camera.